docs/DeveloperGuide.md

```diff
@@ -238,22 +240,72 @@ than using the mouse.
 |v2.0|professor|view calendar of all my events|to see a overview of them|
 
 ## Appendix E: Instructions for Manual Testing
-
+#### Set up
 1. Download the jar file and copy it into an empty folder.
-1. Run the jar file by typing java -jar Pac-2.1.jar after going into the file's home directory folder in command terminal. 
-1. Add an event by typing `event add n/NAME`
-1. View the populated events by typing `event list`
-1. View the populated seminars by typing `seminar list`
-1. Delete an event by typing `delete i/INDEX`
-1. Edit an existing event's name by typing `event editname i/INDEX n/NAME`
+1. Run the jar file by typing java -jar Pac-2.1.jar after going into the file's home directory 
+folder in command terminal.  
+ 
+#### Follow **all-in-one** command for following commands:  
+**Event**
+1. Add an event by typing  
+`event add n/NAME`
+1. View the populated events by typing  
+`event list`
+1. View the populated seminars by typing  
+`seminar list`
+1. Delete an event by typing  
+`delete i/INDEX`
+1. Edit an existing event's name by typing  
+`event editname i/INDEX n/NAME`
 1. Edit an existing event's date and time by typing  
 `event editdatetime i/INDEX d/DATE t/TIME`
 1. Edit an existing event's venue by typing   
-`event editvenue i/INDEX v/VENUE`
+`event editvenue i/INDEX v/VENUE`  
+
+**Calender**
 1. Display calendar by entering  
-`calendar s/SEMESTER ay/YEAR_ONE-YEAR_TWO`
+`calendar s/SEMESTER ay/YEAR_ONE-YEAR_TWO`   
 
+#### Follow **step-by-step** command for following commands:  
```

Comment:
Perhaps for this step-by-step command,  we can also insert in images on the possible ways of using the respective command?

Reply:
it sounds like ug to me, could u elaborate more?